apps/editing-toolkit/editing-toolkit-plugin/global-styles/class-global-styles.php

```diff
@@ -338,6 +338,7 @@ public function enqueue_block_editor_assets() {
 			$version,
 			true
 		);
+		wp_set_script_translations('jetpack-global-styles-editor-script', 'full-site-editing');
```

Comment:
does this kind of thing need to be set for every single script enqueued by the editing toolkit plugin?

Reply:
If the script uses the new i18n functions, this needs to be set up after enqueueing the script. Some more information on JS internalization is available [here](https://developer.wordpress.org/apis/handbook/internationalization/#setting-up-your-plugin-and-theme-to-i18n).

Reply:
thank you!